### ISSUE TITLE
Trading cards sometimes fails

### ISSUE THREAD

**Issue by jscholes** [HearthstoneAccess/HearthstoneAccess#60]
With the latest patch, I'm finding that trading a card consistently doesn't work.  To reproduce:

1. Navigate to a tradable card with the arrow keys, Tab, etc., while having enough mana to trade it.
2. Press T.
3. Press Enter.

#### Expected Result
The screen reader will announce "Trade card?" after pressing T, and then carry out the trade after pressing Enter.

#### Actual Result
More often than not, the screen reader does announce "Trade card?" after pressing T, but pressing Enter seems to do nothing.  The background sound accompanying the trade confirmation continues to play, and I must press Escape to cancel the trade and move on with the game.  This is the outcome the majority of the time, but not in 100% of instances.  I have no idea what causes some trades to work and others not.

**Comment by Timtam** [comment]
I found this to be a matter of speed. The new Hearthstone version is much slower than the previous ones (which is especially noticeable in Battlegrounds btw), and the bug occurs if you press enter too fast before the actual trade question comes up. If you give it time to breath after pressing T is will work constantly, at least thats the situation over here.

**Comment by pitermach** [comment]
The latest version of hearthstone added frame rate options in the game, but the mod doesn't expose them. However, you can manually increase it for now with the trick people used before:
https://acegameguides.com/a-simple-guide-to-unlocking-your-fps-in-hearthstone/ <https://acegameguides.com/a-simple-guide-to-unlocking-your-fps-in-hearthstone/>

If this is what causes that trade bug this should at least help until it's fixed in the mod.

**Comment by alex19EP** [comment]
can reproduce but for me spell sound stops but trade doesn't happen

**Comment by jscholes** [comment]
@pitermach Thanks, I feel like it helped although will have to experiment further.

TL;DR, close the game and Battle.net, open `%localappdata%\Blizzard\Hearthstone\options.txt`, add these two lines at the end and save:

```
targetframerate=60
vsync=1
```

Note that I have no idea what the second setting, `vsync`, is for.

**Comment by jscholes** [comment]
Haven't had this issue for a long time; closing.
